modules/fastcv/test/test_remap.cpp

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+*/
+
+#include "opencv2/ts.hpp"
+#include "opencv2/remap.hpp"
+
+namespace opencv_test { namespace {
+
+class RemapTest : public ::testing::TestWithParam<tuple<int, int, Size>> {
+protected:
+    void SetUp() override {
+        // Generate random source data
+        Size size = get<2>(GetParam());
+        src = Mat(size, get<0>(GetParam()));
+        randu(src, Scalar::all(0), Scalar::all(255)); // Fill with random values
+
+        ASSERT_FALSE(src.empty()) << "Unable to generate the image!";
+
+        // Create map matrices
+        map_x.create(src.size(), CV_32FC1);
+        map_y.create(src.size(), CV_32FC1);
+
+        // Initialize the map matrices
+        for (int i = 0; i < src.rows; i++) {
+            for (int j = 0; j < src.cols; j++) {
+                map_x.at<float>(i, j) = static_cast<float>(src.cols - j); //Flips the image horizonally
+                map_y.at<float>(i, j) = static_cast<float>(i); //Keep y coordinate unchanged
+            }
+        }
+    }
+
+    Mat src, map_x, map_y, dst;
+};
+
+class RemapTestRGBA : public ::testing::TestWithParam<tuple<int, int, Size>> {
+protected:
+    void SetUp() override {
+        // Generate random source data
+        Size size = get<2>(GetParam());
+        src = Mat(size, get<0>(GetParam()));
+        randu(src, Scalar::all(0), Scalar::all(255)); // Fill with random values
+
+        ASSERT_FALSE(src.empty()) << "Unable to generate the image!";
+
+        // Create map matrices
+        map_x.create(src.size(), CV_32FC1);
+        map_y.create(src.size(), CV_32FC1);
+
+        // Initialize the map matrices
+        for (int i = 0; i < src.rows; i++) {
+            for (int j = 0; j < src.cols; j++) {
+                map_x.at<float>(i, j) = static_cast<float>(src.cols - j); //Flips the image horizonally
+                map_y.at<float>(i, j) = static_cast<float>(i); //Keep y coordinate unchanged
+            }
+        }
+    }
+
+    Mat src, map_x, map_y, dst;
+};
+
+TEST_P(RemapTest, accuracy)
+{
+    int type = get<0>(GetParam());
+    int interpolation = get<1>(GetParam());
+
+    // Convert source image to the specified type
+    Mat src_converted;
+    src.convertTo(src_converted, type);
+
+    cv::fastcv::remap(src_converted, dst, map_x, map_y, interpolation);
+
+    // Check if the remapped image is not empty
+    ASSERT_FALSE(dst.empty()) << "Remapped image is empty!";
+
+    cv::Mat remapOpenCV;
+    cv::remap(src_converted, remapOpenCV, map_x, map_y, interpolation);
+
+    cv::Mat diffImage;
+    cv::absdiff(dst, remapOpenCV, diffImage);
+
+    // Calculate the maximum difference
+    double maxVal=0.0;
+    cv::minMaxLoc(diffImage, nullptr, &maxVal);
```

Comment:
L_inf norm does the same: `double maxVal = cv::norm(dst, remapOpenCV, cv::NORM_INF);